### DISCUSSION TITLE
Android

### DISCUSSION THREAD

**Discussion by plue2001** [ihhub/fheroes2#7005] [Ideas]
![Screenshot_2023-04-14-13-05-45~4](https://user-images.githubusercontent.com/130730169/231947371-2ec6932b-f99c-44ee-bf8d-b3839d86e6c6.png)

I wish there were no left and right margins on smartphones (red mark) and it is inconvenient because the direction of the attack is hidden by the finger when attacking.

It would be nice if the arrow direction was added or the sword was enlarged and displayed.

**Comment by ihhub** [comment]
Hi @plue2001 , we have this in plans.

**Reply by plue2001** [reply]
thank you very much!!
